### PR TITLE
Minor addition

### PR DESCRIPTION
**Pull request by haochey**
Add threshold and support_width to input

### CODE REVIEW

File: src/pre_process/m_mpi_proxy.fpp [hunk 1]
```diff
@@ -91,6 +91,7 @@ contains
                 call MPI_BCAST(patch_icpp(i)%${VAR}$, size(patch_icpp(i)%${VAR}$), MPI_DOUBLE_PRECISION, 0, MPI_COMM_WORLD, ierr)
             #:endfor
 
+            call MPI_BCAST(patch_icpp(i)%model%threshold, 1, MPI_DOUBLE_PRECISION, 0, MPI_COMM_WORLD, ierr)
```
Comment (wilfonba) [inline src/pre_process/m_mpi_proxy.fpp:94]: I believe this could be added to the fypp loop above

File: src/simulation/m_global_parameters.fpp [hunk 2]
```diff
@@ -491,6 +491,7 @@ contains
             mono(j)%support = 1
             mono(j)%foc_length = dflt_real
             mono(j)%aperture = dflt_real
+            mono(j)%support_width = 2.5d0
```
Comment (wilfonba) [inline src/simulation/m_global_parameters.fpp:494]: Where does 2.5 come from?

Comment (haochey) [reply src/simulation/m_global_parameters.fpp:494]: It was the hard-coded default for monopole.

File: src/pre_process/m_mpi_proxy.fpp (hunk 1 above)
Comment (haochey) [reply src/pre_process/m_mpi_proxy.fpp:94]: This new input is not an array unlike others in the loop above. I can add it to the loop in line 80 for normal patch inputs if this makes more sense.

File: src/simulation/m_global_parameters.fpp (hunk 2 above)
Comment (sbryngelson) [reply src/simulation/m_global_parameters.fpp:494]: please add a comment for this line. also, why is this a new line? shouldn't the support width have been already present in the code?

File: src/pre_process/m_mpi_proxy.fpp (hunk 1 above)
Comment (sbryngelson) [reply src/pre_process/m_mpi_proxy.fpp:94]: yes

File: src/simulation/m_global_parameters.fpp (hunk 2 above)
Comment (haochey) [reply src/simulation/m_global_parameters.fpp:494]: It's not in the code yet. The new input is the number of characteristic cells the support spans. It uses the new input to calculate the 'sig' which is the actual width of the support.

Comment (sbryngelson) [reply src/simulation/m_global_parameters.fpp:494]: Sounds good to me.

**Comment by sbryngelson** [comment]
Was this closed intentionally?

**Comment by haochey** [comment]
> Was this closed intentionally?

Sorry for any confusions. I closed it because I want to use a forced push to solve a conflict in the root branch. I made a new PR #336.

**Pull request closed without merging**